TODO/building-a-shop-with-sub-second-page-loads-lessons-learned.md

```diff
@@ -37,11 +37,11 @@
 
 
 
-TV-Shows like “Shark Tank” (US), “Dragons’ Den” (UK) or “Die Höhle der Löwen (DHDL)” (GER) offer young startups a one-time opportunity to pitch their products to business magnates in front of a huge audience. The main profit, however, often does not lie in the strategic investments offered by jury members — [only few deals are finalized](http://www.bloomberg.com/news/articles/2014-07-15/shark-tank-do-two-thirds-of-deals-fall-apart) — but rather in the attention raised during the show: a few on-air minutes can already draw hundreds of thousands of new visitors to a website and may boost the base level of activity for weeks, months or even permanently. That is, if the website can take the initial load spike and does not turn down user requests…
+「Shark Tank」（美国），「Dragons’ Den」（英国）或「DieHöhlederLöwen（DHDL）」（德国）等电视节目为年轻初创公司提供了一次机会，将他们的产品推向商业巨头，在众多观众面前。 然而，主要的利润往往不在于陪审团成员提供的战略投资——[只有少数交易会完成](http://www.bloomberg.com/news/articles/2014-07-15/shark-tank-do-two-thirds-of-deals-fall-apart)——而是在电视节目播放期间引起的注意：一些电视时间已经可以吸引成千上万的新访客参观网站，并可以提高活动的基础水平几个星期，几个月甚至永久。 也就是说，如果网站可以抓住初始负载尖峰，并且不拒绝用户请求……
 
-### Availability is not enough — latency is key!
+### 仅仅可用是不够的——延迟是关键！
```

Comment:
Latency is a time interval between the stimulation and response, or, from a more general point of view, a time delay between the cause and the effect of some physical change in the system being observed.